vendors/espressif/boards/ports/ota_pal_for_aws/ota_pal.c

```diff
@@ -367,11 +367,11 @@ uint8_t * otaPal_ReadAndAssumeCertificate( const uint8_t * const pucCertName,
 
     if( ( xResult == CKR_OK ) && ( pucSignerCert != NULL ) )
     {
-        ESP_LOGI( TAG, "Using cert with label: %s OK\r\n", ( const char * ) pucCertName );
+        ESP_LOGI( TAG, "Using cert with label: %s OK", ( const char * ) pucCertName );
```

Comment:
Can we use logging from this repo instead of ESP logs?

Reply:
Updated in https://github.com/aws/amazon-freertos/pull/3287/commits/ecfc6975f2b3910ab1891dd4df3be39e5b887e58